src/main/java/factionsplusplus/FactionsPlusPlus.java

```diff
@@ -39,12 +39,12 @@ public class FactionsPlusPlus extends JavaPlugin {
     private final String pluginVersion = "v" + getDescription().getVersion();
     @Inject private ActionBarService actionBarService;
     @Inject private ConfigService configService;
-    @Inject private DataService dataService;
-    @Inject private Scheduler scheduler;
-    @Inject private CommandService commandService;
+    @Inject private Provider<DataService> dataService;
```

Comment:
Is there an import needed for this? Just wondering

Reply:
yeah, it's there already: import javax.inject.Provider;